packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -143,14 +148,11 @@ export class GuxTimeZonePickerBeta {
   }
 
   private getDefaultZones(): string[] {
-    const defaultZones: string[] = [];
-    if (this.workspaceDefault) {
-      defaultZones.push(this.workspaceDefault);
-    }
-    if (this.localDefault) {
-      defaultZones.push(this.localDefault);
-    }
-    return defaultZones;
+    return [
+      this.workspaceDefault,
+      this.localDefault,
+      this.customDefault
+    ].filter(zone => !!zone);
```

Comment:
what does this line do? `].filter(zone => !!zone);`? For my own understanding. Are you trying to filter out empty values from workspaceDefault, localDefault, and customDefault?

Reply:
Ye so basically if a user has provided a timezone for those properties it will filter those out out say for example you have `gux-timezone-picker-beta` setup like,

```
  <gux-time-zone-picker-beta
    value="Etc/GMT+1"
    workspace-default="Etc/GMT"
    local-default="America/Detroit"
  > 
```

The funtion would then filter out `Etc/GMT` and `America/Detroit` and store those in an array which will be used to display under the default zones list.